hail/src/test/scala/is/hail/HailSuite.scala

```diff
@@ -83,4 +92,205 @@ class HailSuite extends TestNGSuite {
       hc.sparkBackend("HailSuite.withExecuteContext").withExecuteContext(timer)(f)
     }
   }
+
+  def assertEvalsTo(
```

Comment:
I moved several methods from TestUtils into HailSuite so that the execute context would be available.